CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 The changes documented here do not include those from the original repository.
 
+## [2.3.1]
```

Comment:
2.3.0 has not been released yet , so maybe we can join all of this in the 2.3.0 ?